app-lifelike/src/main/java/com/example/lifelike/composable/loggedout/RegConfirmSmsCode.kt

```diff
@@ -15,19 +15,19 @@ interface RegConfirmSmsCode {
         @ExperimentalFoundationApi
         @Composable
         fun Content(onNext: () -> Unit) {
-            val code = state {
+            var code = remember {
```

Comment:
This will definitely break this screen, value updates now won't trigger a recompose. I think you need `mutableStateOf`.